app-foundation/3-networks-extension/README.md

```diff
@@ -0,0 +1,26 @@
+# 3-networks-extension
+
+This is an additional Terraform configuration for [example-foundation 3-Networks](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/3-networks) and is used to set up the subnets, additional firewall rules for the Bank of Anthos example application across dev/non-prod/prod environments.
+
+## Prerequisites
+
+1. 0-bootstrap executed successfully.
+1. 1-org executed successfully.
+1. 2-environments/envs/ executed successfully.
+
+## Order of Execution
+
+1. Run merge_3-networks.sh
```

Comment:
nit name had changed. Lets call out make docker_network_prepare command and how to execute `cd .././ && make docker_network_prepare && cd -`